src/shared/views/ChangeDetection/Tile.js

```diff
@@ -199,38 +179,14 @@ export class _Tile extends React.PureComponent<Props> {
             );
         }
         const imageSource = this.getImgSource();
-        let comp;
-
-        if (this.getOsmBuildingsUrl() !== undefined) {
-            comp = (
-                <ImageBackground
-                    style={styles.buildingStyle}
-                    source={this.getOsmBuildingsUrl()}
-                >
-                    <View
-                        style={[
-                            styles.tileOverlay,
-                            { backgroundColor: overlayColor },
-                        ]}
-                        key={`view-${taskId}`}
-                    >
-                        {animatedRows}
-                    </View>
-                </ImageBackground>
-            );
-        } else {
-            comp = (
-                <View
-                    style={[
-                        styles.tileOverlay,
-                        { backgroundColor: overlayColor },
-                    ]}
-                    key={`view-${taskId}`}
-                >
-                    {animatedRows}
-                </View>
-            );
-        }
+        const comp = (
```

Comment:
Is it worth having this variable here, since it always has the same value? Maybe inline this component into the main one a few lines down for clarity?

Reply:
Hey, yes good comment. I moved it down.